src/jit/gentree.h

```diff
@@ -2887,6 +2887,7 @@ struct GenTreeCall final : public GenTree
         InlineCandidateInfo * gtInlineCandidateInfo;
         void *  gtStubCallStubAddr;             // GTF_CALL_VIRT_STUB - these are never inlined                
         CORINFO_GENERIC_HANDLE compileTimeHelperArgumentHandle; // Used to track type handle argument of dynamic helpers
+        void * gtDirectCallAddress; // Used to pass direct call address between lower and codegen
```

Comment:
This field is union with `gtInlineCandidateInfo` that is set to null and cloned already - the assumption is that `gtInlineCandidateInfo` is not going to be used after lowering.